### DISCUSSION TITLE
Fixtures data were empty

### DISCUSSION THREAD

**Discussion by radim-suk** [georgedouzas/sports-betting#37] [General]
Hello 

I tried your script yesterday for the first time, with default (example) config files and It worked flawlesly - it gives me 6 rows of fixtures (6 soccer matches).
If I run it today, it just outputs "Fixtures data were empty" after running "sportsbet dataloader fixtures -d dataloader_config.py".

I tried different odds_types and leagues in dataloader_config.py but still, but still no odds for today.
I tried to delete all data and retrain again, i tried to delete everything and download whole library from github again, but still the same Error.

Is this normal behavior or am I doing something wrong?

Thanks
Radim

**Comment by georgedouzas** [comment]
Hi Radim,

Yes, it is normal behavior, not really an error, since fixtures are supposed to be upcoming matches. If no upcoming matches are found, i.e. no data about them is found, then fixtures data are empty. Usually, the data about fixtures are available a few days before the start of the matches.
